dd-java-agent/instrumentation/grizzly-2/src/main/java/datadog/trace/instrumentation/grizzly/GrizzlyHttpHandlerInstrumentation.java

```diff
@@ -79,6 +84,11 @@ public static AgentScope methodEnter(@Advice.Argument(0) final Request request)
       final AgentScope scope = activateSpan(span, false);
       scope.setAsyncPropagation(true);
 
+      if (Config.get().isEmitServerTimingContext() && response != null) {
```

Comment:
I think it is worth documenting somewhere why we add `traceparent` header to the _response_. Original meaning is to add it to the outgoing request.

Reply:
Server-Timing is only ever a response header.  But, yes, this is funny.

Reply:
I know that, you know that. But a developer reading this in two months will be confused. Please add background info

Reply:
Will add commentary in the config entry rather than all around the code base.